File: playground/BOJ/boj_1264/boj_1264.js
/*
----------------------------------------------
Sub : [BOJ] 모음의 개수
Link: https://www.acmicpc.net/problem/1264
Tag : JS, 
Memo
----------------------------------------------
*/

const fs = require('fs');

// 입력 파일 읽기
const input = fs.readFileSync('/dev/stdin').toString().trim().split('\n');

// 모음 목록
const vowels = 'aeiouAEIOU';

// 모음 개수 세는 함수
function countVowels(line) {
    let count = 0;
    for (let char of line) {
        if (vowels.includes(char)) {
            count++;
        }
    }
    return count;
}

// 각 줄마다 모음 개수 세기
input.forEach(line => {
    if (line === '#') return;
    console.log(countVowels(line));
});

File: playground/boj_1264/boj_1264.py
'''
----------------------------------------------
Sub : [BOJ] 모음의 개수
Link: https://www.acmicpc.net/problem/1264
Tag : Python, 
Memo
----------------------------------------------
'''

while True:
    n = input()
    if(n == '#'): 
        break
    else :
        sum = 0
        sum += n.count('a') + n.count('e') + n.count('i') + n.count('o') + n.count('u')
        sum += n.count('A') + n.count('E') + n.count('I') + n.count('O') + n.count('U')

    print(sum)
